demo/src/screens/componentScreens/MarqueeScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
+import React, {Component} from 'react';
+import {View, Text, Marquee} from 'react-native-ui-lib';
+
+export default class MarqueeScreen extends Component { 
+  renderRow() {
+    return (
+      <View row>
+        {_.times(8, i => {
+          return <View key={i} style={{backgroundColor: '#FF10F0', width: 20, height: 6, marginRight: 10}}/>;
+        })}
+      </View>
+    );
+  }
+  
+  render() {
+    return (
+      <>
+        <Text text50 grey10 margin-20>Marquee</Text>
+        <View marginH-50 style={{borderWidth: 1}}>
+          <Marquee start={50} end={50} duration={9000}>
```

Comment:
We can remove the `start` and `end` props here

Reply:
Done